assets/js/callback.js

```diff
@@ -1 +1,37 @@
-alert("authentication callback complete.")
+// alert("authentication callback complete.")
+
+function redirectUser() {
+
+  let redirectUrl
+
+  const params = new URLSearchParams(window.location.search)
```

Comment:
This seems to be constructed based on how t-pen.org works, but I think you want it to look more like https://github.com/CenterForDigitalHumanities/DLA: 

```js
auth.js:35

const getReferringPage = () => {
    try {
        return b64toUrl(location.hash.split("state=")[1].split("&")[0])
    } catch (err) {
        return false
    }
}
```